### ISSUE TITLE
Galerkin Neural ODEs

### ISSUE THREAD

**Issue by ChrisRackauckas** [SciML/DiffEqFlux.jl#290]
https://arxiv.org/abs/2002.08071

**Comment by Raj0828** [comment]
@ChrisRackauckas Hi Chris! I do not have experience in contributing to open source project and therefore when I saw the "good first issue" mention, I clicked on this. I have some background in numerical ODE and PDE. What do you have in mind for this issue? Is it implementation of some of the ideas described in the arxiv paper?

**Comment by ChrisRackauckas** [comment]
Yes, take a look at how NeuralODE is defined, and use the ideas from the paper to implement a GalerkinNeuralODE struct.

**Comment by RajeshMishra95** [comment]
Alright! I will give it a shot. Thanks!

**Comment by Biswajitghosh98** [comment]
@ChrisRackauckas I'll start working on it

**Comment by ChrisRackauckas** [comment]
@Biswajitghosh98 are you still claiming this?

**Comment by Biswajitghosh98** [comment]
@ChrisRackauckas I've been a little engaged in college work lately. If anyone wants to start working, then I'll catch up later

**Comment by neelesh17** [comment]
@ChrisRackauckas Hi Chris, I am a GSOC2021 aspirant. I have some experience in contributing to open source project and numerical ODE and PDE. I would like to give this issue a try.

**Comment by ChrisRackauckas** [comment]
Alright great!
